src/VisualStudio/Core/Def/Implementation/IntellisenseControls/IntellisenseTextBoxViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.IntellisenseControls
+{
+    internal readonly struct IntellisenseTextBoxViewModel
+    {
+        public readonly IVsTextView VsTextView;
+        public readonly IWpfTextView WpfTextView;
```

Comment:
how is this Model?  I assuem this is 'Model' in hte 'MVC' sense. but this seems to be entirely about the view.

Reply:
This is what we need to create an instance of the IntellisenseTextBox control. Do you think IntellisenseTextBoxView is a better name for it?